Jenkinsfile

```diff
@@ -200,8 +200,20 @@ pipeline {
             }
 
             post {
+                always {
+                    jacoco(
+                          execPattern: '**/*.exec',
+                          classPattern: '**/target/classes',
+                          sourcePattern: '**/src/main/java,**/generated-sources/protobuf/java,**/generated-sources/assertj-assertions,**/generated-sources/sbe',
+                          exclusionPattern: '**/io/zeebe/gateway/protocol/**,'
```

Comment:
Why are we not excluding all generated code, but only some generated code (e.g. all SBE `*Encoder.class`)?

Reply:
This is not as easy as it looks. Basically the exclusions are based on file patterns in the `classes` folder, not the source folders. The source folders are easy to spot, but then all classes get lumped in together. 

Exclusion by packages also doesn't work, because the generated code shares packages with actual code. I had several iterations and this was what covered like 90% of generated code.

Reply:
Oh, that's disappointing, I would've expected we could just ignore specific source folders =/ Let's start with that then and if it's an issue we can look into more complex solutions